specs/Menu.md

```diff
@@ -1,731 +1,669 @@
-| iconOnly             | &#x274C;               | &#x274C;         | &#x274C;              |
-| inSubmenu            | &#x274C;               | &#x274C;         | &#x274C;              |
-| index                | &#x274C;               | &#x274C;         | &#x274C;              |
-| indicator S          | &#x274C;               | &#x274C;         | &#x274C;              |
-| itemPosition         | &#x274C;               | &#x274C;         | &#x274C;              |
-| itemsCount           | &#x274C;               | &#x274C;         | &#x274C;              |
-| menu S               | &#x274C;               | &#x274C;         | &#x274C;              |
-| menuOpen             | &#x274C;               | &#x274C;         | &#x274C;              |
-| onActiveChanged      | &#x274C;               | &#x274C;         | &#x274C;              |
-| onBlur               | &#x274C;               | &#x274C;         | &#x274C;              |
-| onClick              | &#x274C;               | &#x274C;         | &#x274C;              |
-| onFocus              | &#x274C;               | &#x274C;         | &#x274C;              |
-| onMenuOpenChange     | &#x274C;               | &#x274C;         | &#x274C;              |
-| pills                | &#x274C;               | &#x274C;         | &#x274C;              |
-| pointing             | &#x274C;               | &#x274C;         | &#x274C;              |
-| primary              | &#x274C;               | &#x274C;         | &#x274C;              |
-| secondary            | &#x274C;               | &#x274C;         | &#x274C;              |
-| styles               | &#x274C;               | &#x274C;         | &#x274C;              |
-| underlined           | &#x274C;               | &#x274C;         | &#x274C;              |
-| variables            | &#x274C;               | &#x274C;         | &#x274C;              |
-| vertical             | &#x274C;               | &#x274C;         | &#x274C;              |
-| wrapper S            | &#x274C;               | &#x274C;         | &#x274C;              |
-
-### Notable things
-
-Based on my recommendations, the MenuItem ends up doing a lot of work compared to the Menu itself. The MenuItem would be responsible for managing whether or not its expanded, has a submenu, and any other state it might have.
-
-## DOM Structure
-
-### HTML DOM structure
-
-```HTML
-<ul role="menu">
-    <li role="menuitem">{"Item one"}</li>
-    <hr />
-    <li role="menuitem">{"Item two"}</li>
-    <li role="menuitem">
-        {"Item Three"}
-        <ul role="menu">
-            <li role="menuitem">{"SubItem one"}</li>
-            <li role="menuitem">{"SubItem two"}</li>
-        </ul>
-    </li>
-</ul>
+v0 uses the [OSS Popper.js library](https://popper.js.org/) while v7 uses a component based implementation `CalloutContent`. Below we provide the results of testing common positioning boundary/edge cases between the two.
+
+// TODO compare boudary/edge cases in positioning
```

Comment:
Also consider dual-screen support (#12785). Double check whether it is currently a requirement.